layers/meta-balena-beaglebone/recipes-kernel/linux/linux-beagleboard_%.bbappend

```diff
@@ -60,6 +60,6 @@ BALENA_CONFIGS[hm3301] = " \
 # still be aufs, so we need to include the aufs driver further
 # for them, as per the internal thread
 # https://www.flowdock.com/app/rulemotion/resin-devices/threads/K2TQiSUfNDqBT5Ih6cciNI2d9QJ
-BALENA_CONFIGS_append_beaglebone = " aufs"
-BALENA_CONFIGS_append_beaglebone-green = " aufs"
-BALENA_CONFIGS_append_beaglebone-green-wifi = " aufs"
+BALENA_CONFIGS:append_beaglebone = " aufs"
```

Comment:
New syntax conversion here also is needed

Reply:
I'm actually on that!